packages/core/admin/admin/src/components/NpsSurvey/index.js

```diff
@@ -124,6 +126,46 @@ const NpsSurvey = () => {
   const { formatMessage } = useIntl();
   const { npsSurveySettings, setNpsSurveySettings } = useNpsSurveySettings();
   const [isFeedbackResponse, setIsFeedbackResponse] = React.useState(false);
+  const toggleNotification = useNotification();
+
+  const { mutate, isLoading } = useMutation(
+    async (form) => {
+      const res = await fetch('https://analytics.strapi.io/submit-nps', {
```

Comment:
I'm just wondering if we could / should store this in a variable based on the environment. Otherwise in development we are going to pollute the production nps data.

Reply:
Good point! I'll ask Yannis